xapian-applications/omega/templates/query

```diff
@@ -153,3 +153,4 @@ $map{$cgilist{B},<input type=hidden name=B value="$html{$_}">}
 </body>
 </html>
 $log{query.log}
+$log{search.log,$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}	$csv{$query,1}	$hitlist{$id$.}	$topdoc}
```

Comment:
Similar comments to the previous logging. There's also something odd (and not necessarily correct) about using `$csv{}` when you're generating a TSV. TSV is flimsily specified, perhaps even more so than CSV :-(

You can avoid the trailing `,` in the hitlist by checking `$hit` against `$sub{$last,1}`.